src/Widgets/TerminalWidget.vala

```diff
@@ -293,12 +314,6 @@ namespace Terminal {
                 }
             }
 #endif
-
-            /* Disable bell if necessary */
-            audible_bell = Application.settings.get_boolean ("audible-bell");
-
-            /* Cursor shape */
-            set_cursor_shape ((Vte.CursorShape) Application.settings.get_enum ("cursor-shape"));
```

Comment:
I get pulling this out into a new function but the way it is written now, being called in the constructor, those update functions won't be called when Application.settings.changed is signaled. Is that alright?

Reply:
No they should only be called when the specific settings change that require a new set here